### ISSUE TITLE
I goy "failed to get game date from server" massege, but i dont riceve any email from Nientic.

### ISSUE THREAD

**Issue by forestlin98** [finndev/PokeBuddy#1167]
i got this massege 2 days before.
thanks for help : D

**Comment by kwafer1** [comment]
ban

**Comment by forestlin98** [comment]
@kwafer1 It's a permanet ban?

**Comment by macoy13** [comment]
Me too. Is it a permanent ban? Haven't received email from Niatic.

**Comment by PSUolive** [comment]
Welcome to the club 
You have been Perm Ban

**Comment by pokemonloverx333** [comment]
perma ban its the 43 thread about the same shit , why is no one reading previous issues...

**Comment by Demolition1986** [comment]
Well i feel lucky, 1 bot account created days ago with necro bot got lv 24 or 25 and with app for teleport i got a lot of rare pokemon teleporting for all country in the world, my main account lv 29 also both account not banned yet.. So i feel lucky.

**Comment by lilililo920** [comment]
Niantic won't send u email about banned account,  they just do it & forget it.

**Comment by Abismo** [comment]
I don't know if it's a perma ban or not, but probably yes. I got this message on 3 accounts and only one of them is still banned, for 3+ days now so I guess it's permanent.

**Comment by Kokainium** [comment]
I got it too :o and i wasnt even using bot :D

**Comment by Kokainium** [comment]
Cant even log out :o

**Comment by Abismo** [comment]
Go to your app settings and delete data from PoGo to log out.

**Comment by Kokainium** [comment]
@abismo did u say that u have got that one before but now u are able to get in?

**Comment by Abismo** [comment]
If the account is banned you can't log in. But you can do it with another account.

**Comment by OsOCooL** [comment]
I have the same thing three days ago, no email. I emailed and they replied within an hour. If you have the "failed to get game data" just consider your account gone. 
![screen shot 2016-08-15 at 11 17 36 am](https://cloud.githubusercontent.com/assets/20901750/17680038/fa6a981a-62d9-11e6-8ea8-6502f2f54fe7.png)

**Comment by Kokainium** [comment]
To be honest i hate this game... doesn't really make sense Cus u cant get all pokemons no matter where u live :s

**Comment by Abismo** [comment]
You can do it if you live in a big city like NY or LA, otherwise it's impossible.

**Comment by Kokainium** [comment]
And i live en denmark 😑

**Comment by Kokainium** [comment]
In

**Comment by Rab89** [comment]
you do know you can edit posts :laughing:

**Comment by Kokainium** [comment]
Not on my Phone:p

**Comment by DatHacksGaming** [comment]
Please mark this as closed >.> there are 30 other threads about being banned already lol

**Comment by ravdesai09** [comment]
4 days before one account banned and today again mine 2 account banned... RIP lol.. may be now out from game from me..

**Comment by staticvg** [comment]
i got that message on my other account earlier today re tried a hour ago and i was back in working fine and everything.

**Comment by Demolition1986** [comment]
I have 2 account fine too, let see these days.

**Comment by cswxin** [comment]
I got the same thing this morning :(

**Comment by Abismo** [comment]
@Demolition1986 My main account was banned, then unbanned. That happened 4 days ago. Now it's banned again, so be careful :)

**Comment by macoy13** [comment]
Is there any chance that they will uplift the ban? I read some comments that some perm bans are uplifted in 12hrs, 1 day or 2 days. I haven't receive an email from Niantic.

**Comment by Abismo** [comment]
Yeah, there's always a chance, but after 3 days I wouldn't wait for it any more. They won't email you nor say anything. Officially you aren't banned, you just can't get data from the server, but we know what that means.

**Comment by Kokainium** [comment]
Mine is still down 😂😋

**Comment by majdak79** [comment]
@Demolition1986 dont spread BS. go to an arena and fight with your account and you are done as everyone. just if you dont go arenas, you might be fine, but not for sure. so far everyone got banned by using pokebuddy

**Comment by Abismo** [comment]
I'm using NecroBot right now, since PokeBuddy isn't working for me. And if I get banned again... Who cares? There's only pidgey and ratata in my area, I wouldn't play without spoofing anyway.

**Comment by Kokainium** [comment]
Same here

**Comment by macoy13** [comment]
@majdak79 how can I go to an arena if I can't open my account? I still got this message "failed to get game date from server"

**Comment by ShupengHu** [comment]
My main account was bannned two days ago.  It seems to be perm ban because I couldnt log in unitil now. But never mind, I just registered a new account and played again with bot until that account was banned.

**Comment by Demolition1986** [comment]
@abismo Same here, in my place just ratata, so spoff is the best way to go. And i save my location cords in same place all the time to farm different pokemon.. Around 3 pokestops
